MAiD - Case Management App/force-app/main/default/layouts/Case-MAiD Case Layout.layout-meta.xml

```diff
@@ -427,11 +436,6 @@
             <actionType>QuickAction</actionType>
             <sortOrder>5</sortOrder>
         </platformActionListItems>
-        <platformActionListItems>
```

Comment:
@RanadheerRG, please add this quick action back, unless Lara asked us to remove it from the page layout.

Reply:
It's already added in line 404